.github/workflows/ci_full.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       matrix:
         os: [ubuntu-latest]
         python-version:
-          - "3.10"
+          - "3.11"
```

Comment:
Python 3.12 is planned to be release today, 2023-10-02 (probably US time). We can add the support for 3.12 too.

Monitor at:

* https://www.python.org/downloads/
* https://github.com/actions/python-versions/releases (the CI-enabled versions).